### ISSUE TITLE
Is there anyway to record the IP address of client when punch in or out? Thanks a lot.

### ISSUE THREAD

**Issue by solohsi** [kimai/kimai#4079]
### Describe the problem

I want to record the IP address of client.

### Describe the solution you'd like

Is there anyway to record the IP address of client when punch in or out? Thanks a lot.

**Comment by kevinpapst** [comment]
No. The Kiosk plugin has such a function, but nothing else right now out-of-the-box.

**Comment by solohsi** [comment]
Thanks a lot, and then is there any plan to support this function?

**Comment by solohsi** [comment]
Or, could you please help me to point out which class and file should I deal with to develop it by myself? I try to find it but failed. Thanks a lot
